File: java/Pacote.java
package Correios;


public class Pacote {

	private int id;
	private String CPFdestinatario;
	
	public Pacote(){
		
	}
	
	public Pacote (int id, String CPFdestinatario) {
		setId(id);
		setCPFdestinatario(CPFdestinatario);
		
	}
	
	public void setId(int id) {
		this.id = id;
	}
	
	public int getId() {
		return id;
	}
	
	public void setCPFdestinatario(String CPFdestinatario) {
		this.CPFdestinatario = CPFdestinatario;
	}
	


	
	        
	
}

File: python/pacote.py
class Pacote:
    id: int
    cpf_destinatario: str

    def __init__(self, id: int, cpf_destinatario: str):
        self.setId(id)
        self.set_cpf_destinatario(cpf_destinatario)

    def setId(self, id: int):
        self.id = id

    def getId(self):
        return self.id

    def set_cpf_destinatario(self, cpf_destinatario: str):
        self.cpf_destinatario = cpf_destinatario
